Create terse notes for meeting 2022-04-04 (#251)
InnerSourceCommons/innersourcecommons.org: content/about/board/2022-04-04.md

---
layout: page
show_meta: false
title: '2022-04-04'
---

### Date and Time of Meeting

2022-04-04, 7 p.m. - 8 p.m. CET / 12 p.m. - 1 p.m. CDT

* Call to order 7:05 p.m.
* Adjourned 8:11 p.m.
* Next meeting 19 April 2022 18:00 UTC - **to be confirmed** - 

### Roll Call

#### Directors Present  

- Johannes Tigges (Assistant Secretary)
- Russell Rutledge (Secretary)
- Max Capraro (Assistant Treasurer)
- Danese Cooper (Chair, Proxy for the second half via Isabel)
- Georg Grütter (Vice President)
- Isabel Drost-Fromm (President)

- Daniel Izquierdo Cortazar (Proxied via Danese Cooper)

#### Directors Absent

- Jacob Green
- Cedric Williams (Treasurer)

#### Members Present
- Clare Dillon (Executive Director)

> ### Votes Taken

Reviewer: ```suggestion
### Votes Taken
- Danese moved to
  - a) extend the number of directors to 11 and include Russel Rutledge and Maximilian Capraro on the board or
  - b) keep the current size of the board of 9 and involve Russel Rutledge and Maximilian Capraro as officers not serving on the board.
  - Option a) was adopted (5 votes), option b) was rejected (2 votes).
```

Is this a good way to represent the vote?

Reviewer: For the record, we will not merge my 2nd suggestion (mention the vote) because it was technically not in accodance with our bylaws (Section 8.06)

Reviewer 2: We realized after that vote was taken that we're not allowed to increase the size of the board on our own - only the members can do that (see [Section 3.02](https://docs.google.com/document/d/109XWFL_MypH9V2gMd8my0YFzxOQkwJTF/edit)).  Because of that, we don't feel that the vote was a valid action.  Suggest leaving it out of these official notes as we are not giving it heed.

> - Danese moved to formally suspend board rotation mechanism until is is reinstated after discussion around possible needed changes. (4/2/1) (Y/N/A) 

Reviewer: ```suggestion
- Danese moved to formally suspend board rotation mechanism and reinstate it after consensus about potential changes has been reached. (4/2/1) (Y/N/A) 
```
